tests/functional/container/test_container.py

```diff
@@ -320,6 +320,52 @@ def test_create_with_unknown_storage_policy(self):
         data = self.json_loads(resp.data)
         self.assertEqual(data["status"], 480)
 
+    def test_cycle_properties(self):
+        params = self.param_ref(self.ref)
+
+        def get_ok(expected, expected_version):
+            resp = self.request('POST', self.url_container('get_properties'),
+                                params=params)
+            self.assertEqual(resp.status, 200)
+            body = self.json_loads(resp.data)
+            self.assertIsInstance(body, dict)
+            self.assertIsInstance(body.get('properties'), dict)
+            self.assertDictEqual(expected, body['properties'])
+            self.assertEqual(expected_version,
+                             body['system']['sys.m2.version'])
+
+        def del_ok(keys):
+            resp = self.request('POST', self.url_container('del_properties'),
+                                params=params, data=json.dumps(keys))
+            self.assertEqual(resp.status, 200)
```

Comment:
off topic: it's unfortunate that this request returns 200 and an empty JSON object instead of 204 and no body.